WcaOnRails/app/webpacker/components/PostingCompetitions/index.js

```diff
@@ -92,10 +92,14 @@ function PostingCompetitionsIndex({
                 {' '}
                 <CountryFlag iso2={c.country_iso2} />
               </Header.Subheader>
+              <Header.Subheader>
+                {`Submission Timestamp: ${c.results_submitted_at}`}
+                {/* TODO: Convert the above timestamp to a readable way */}
+              </Header.Subheader>
```

Comment:
If the competitions are shown with the "oldest" pending first, do we even need to display the timestamp?

Reply:
Yes according to @SAuroux. I too think it will be good so that we can easily know from dashboard when is the oldest pending results.

Reply:
@viroulep @danieljames-dj My thought here was that this often influences in what sets we are posting results (when we are not able to post all at once). Maybe instead of displaying raw results_submitted_at we can rather just show "Submitted N hours ago"?

Reply:
Yes it's possible, done. 👍